src/Polly.Shared/Monkey/MonkeyEngine.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Threading;
+using Polly.Utilities;
+
+namespace Polly.Monkey
+{
+    internal static partial class MonkeyEngine
+    {
+        private static Random rand = new Random();
```

Comment:
Is this being used anymore since `RandomGenerator.GetRandomNumber()` was added?

Reply:
Nope will remove